README.md

```diff
@@ -67,3 +67,120 @@ teracy-dev:
         branch: develop
       require_version: ">= 0.4.0-SNAPSHOT"
 ```
+
+## Supported configuration
+These are the supported configuration in teracy-dev-core: `variables`, `vagrant`, `default`, `node`.
+
+
+### variables
+`variables` which is the environment variables must be defined to use `teracy-dev`. All variables
```

Comment:
`variables` is used to define dynamic config values. You can define variable keys with environment variables.